_data/events.yml

```diff
@@ -317,3 +317,126 @@
+  expiration: '20241010'
+  host: Revolut
+  speaker: Julia Babahina, Co-Founder / Director of Women Coding Community and a Lead
+    Risk Management Specialist at
+  time: 6:00 PM UTC
+  image:
+    path: /assets/images/events/event-writing-club.jpeg
+    alt: Writing Club - applying the Pyramid Principle in your writing  cover photo
+  link:
+    path: https://www.meetup.com/women-coding-community/events/303682675/?eventOrigin=group_events_list
+    title: View meetup event
+    target: _target
+
+- title: Design Patterns Course
+  description: Join Women Coding Community for our next session on design patterns!
+  category_style: tech-talk
+  category_name: Tech Talk
+  date: Wed, Oct 16, 2024
+  expiration: '20241016'
+  host: Revolut
+  speaker: Julia Babahina, Co-Founder / Director of Women Coding Community and a Lead
+    Risk Management Specialist at
+  time: 6:00 PM UTC
+  image:
+    path: /assets/images/events/tech-talk-20241016.webp
+    alt: Design Patterns Course cover photo
+  link:
+    path: https://www.meetup.com/women-coding-community/events/303526166/?eventOrigin=group_events_list
+    title: View meetup event
+    target: _target
+
+- title: 'Book Club: Mindset'
+  description: "\"After decades of research, world-renowned Stanford University psychologist\
+    \ Carol S. Dweck, Ph.D., discovered a simple but groundbreaking idea: the power\
+    \ of mindset. In this brilliant book, she shows how success in school, work, sports,\
+    \ the arts, and almost every area of human endeavor can be dramatically influenced\
+    \ by how we think about our talents and abilities. People with a fixed mindset\
+    \ \u2014 those who believe that abilities are fixed \u2014 are less likely to\
+    \ flourish than those with a growth mindset \u2014 those who believe that abilities\
+    \ can be developed. Mindset reveals how great parents, teachers, managers, and\
+    \ athletes can put this idea to use to foster outstanding accomplishment.\""
+  category_style: book-club
+  category_name: Book Club
+  date: Thu, Oct 24, 2024
+  expiration: '20241024'
+  host: Revolut
+  speaker: Julia Babahina, Co-Founder / Director of Women Coding Community and a Lead
+    Risk Management Specialist at
+  time: 6:00 PM UTC
+  image:
+    path: /assets/images/events/book-club-20241024.webp
+    alt: 'Book Club: Mindset cover photo'
+  link:
+    path: https://www.meetup.com/women-coding-community/events/303700311/?eventOrigin=group_events_list
+    title: View meetup event
+    target: _target
+
+- title: 'Cloud Career Journeys: DevOps Engineering'
+  description: Join Women Coding Community for the launch of an exciting series on
+    exploring cloud careers! The series will feature expert speakers who are excelling
+    in various cloud-based roles.
+  category_style: tech-talk
+  category_name: Tech Talk
+  date: Tue, Oct 29, 2024
+  expiration: '20241029'
+  host: Airat Yusuff
+  speaker: Oyinkansola Aremu
+  time: 7:00 PM UTC
```

Comment:
Want to double check the event time with the speaker - as the clock will change on 27th Oct

Reply:
I'll do this as well, I reckon the time would still remain 7PM but would be good to confirm